src/provider.rs

```diff
@@ -50,4 +96,13 @@ impl TimeZoneProvider for NeverProvider {
     fn get_named_tz_offset_nanoseconds(&self, _: &str, _: i128) -> TemporalResult<TimeZoneOffset> {
         unimplemented!()
     }
+
+    fn get_named_time_zone_transition(
```

Comment:
Should we be consistent on the naming? We use `time_zone` here, but `tz` on the rest of the methods.

Reply:
Yeah, I can change it back for consistency. I'd really like to bikeshed the naming and methods at some point. `tz` really isn't as clear as I'd like it to be, but I'm not sure I'm sold on whether it should be `time_zone`, `timezone` or something else.